### PR TITLE
Add Park.io

### PR DESCRIPTION
**Pull request by dleshem**
Alexa rank: [57,838](https://www.alexa.com/siteinfo/park.io)

### CODE REVIEW

File: _data/domains.yml [hunk 1]
```diff
@@ -438,6 +438,13 @@ websites:
       - software
     doc: https://www.pairdomains.com/kb/posts/345
 
+  - name: Park.io
+    url: https://park.io/
+    img: parkio.png
+    tfa:
+      - software
+    doc: https://park.io/support
```
Comment (Carlgo11) [inline _data/domains.yml:446]:
```suggestion
```

File: _data/domains.yml [hunk 2]
```diff
@@ -438,6 +438,13 @@ websites:
       - software
     doc: https://www.pairdomains.com/kb/posts/345
 
+  - name: Park.io
+    url: https://park.io/
+    img: parkio.png
+    tfa:
+      - software
```
Comment (Carlgo11) [inline _data/domains.yml]: Please don't use the software tag. Use proprietary or totp depending what the site supports.
See [CONTRIBUTING.md](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#new-sites)

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
It's always a good idea to see if the image can be compressed better even if it's below 2.5KB
park.io

File: _data/domains.yml (hunk 1 above)
Comment (dleshem) [reply _data/domains.yml:446]: This link points to a FAQ page with the following question: "Can I use multi-factor authentication to add security to my Park.io account?".

Unfortunately there's no way to link directly to this question (no anchor in HTML).

I suggest we keep the link - it's better than nothing...

File: _data/domains.yml (hunk 2 above)
Comment (dleshem) [reply _data/domains.yml]: Park.io supports Authy accounts (7-digit codes). It won't work with, say, Google Authenticator.

Do you consider this proprietary or standard TOTP?

Comment (Carlgo11) [reply _data/domains.yml]: That's a proprietary system (aka non RFC-6238).

**Comment by phallobst** [comment]
Since the support page mentions "Authy", the tfa method should be "totp" instead, shouldn't it?

**Comment by Carlgo11** [comment]
@phallobst no, Authy uses a proprietary system.

**Comment by phallobst** [comment]
So the method itself is time-base one time passwords (aka "totp"), but since Authy is not initialized by scanning a QR code (or entering the secret manually), you cannot replace it by any other "totp" application, thus it counts as "proprietary". Did I understand that correctly?

**Comment by Carlgo11** [comment]
That’s correct.
The totp tag really means that the site uses the RFC-6238 standard but that would be too complex for most people to understand.

**Comment by phallobst** [comment]
They say that Authy is compliant, just not using the defaults:
https://www.twilio.com/docs/authy/api/one-time-passwords
https://twitter.com/authy/status/498244613766139904

**Comment by kmpoppe** [comment]
@phallobst It's far too early for me to read through the RFC and see if they **really** are RFC compliant, but your assessment (no other Applications can read an Authy code, thus you **must** use the Authy App) is quite correct, that's the literal definition of proprietary.

File: _data/domains.yml (hunk 1 above)
Comment (dleshem) [reply _data/domains.yml:446]: @Carlgo11 Can you confirm leaving the doc link as-is, lacking a better option?

Comment (Carlgo11) [reply _data/domains.yml:446]: Yeah it’s fine.

**Review by Carlgo11 — APPROVED** [review]

**Review by kmpoppe — APPROVED** [review]

**Pull request merged**